frontend/src/libs/components/stepper/project-name-step/project-name-step.tsx

```diff
@@ -47,7 +37,7 @@ const ProjectName: React.FC<Properties> = ({ onSubmit }: Properties) => {
           type="text"
           label=""
```

Comment:
Name the label, use the example prop to hide it: 
![image](https://user-images.githubusercontent.com/106602282/228667872-c2413526-4077-4a91-8aa1-f700a25a002c.png)

Reply:
> Name the label, use the example prop to hide it: ![image](https://user-images.githubusercontent.com/106602282/228667872-c2413526-4077-4a91-8aa1-f700a25a002c.png)

Reply:
Yeah,forgetting to do this in previous comment